model/cloud/openrouter.go

```diff
@@ -0,0 +1,51 @@
+package cloud
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/sashabaranov/go-openai"
+)
+
+type openRouterProvider struct {
+	// client is the client to query the LLM model.
+	client *openai.Client
+}
+
+// OpenRouterToken is the "OPENROUTER_TOKEN" environment variable.
+var OpenRouterToken = os.Getenv("OPENROUTER_TOKEN")
```

Comment:
Needs to be a parameter. The Env variable should be resolved on the "cmd" level not in the model/router/... code.